release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -873,6 +873,14 @@ Available as a Technology Preview, you can use the sigstore project with {produc
 Enhancements have been added to the process for xref:../nodes/nodes/nodes-nodes-adding-node-iso.adoc#adding-node-iso[adding worker nodes to an on-premise cluster] that was introduced in {product-title} 4.17.
 With this release, you can now generate Preboot Execution Environment (PXE) assets instead of an ISO image file, and you can configure reports to be generated regardless of whether the node creation process fails or not.
 
+[id="ocp-release-notes-nodes-select-kernel-arguments_{context}"]
+==== Node Tuning Operator properly selects kernel arguments
+The Node Tuning Operator can now properly select kernel arguments and management options for Intel and AMD CPUs. (link:https://issues.redhat.com/browse/OCPBUGS-43664[*OCPBUGS-43664*])
+
+[id="ocp-release-notes-nodes-default-container-runtime_{context}"]
+==== Default container runtime is not always set properly
+The default container runtime that is set by the cluster Node Tuning Operator will always be inherited from the cluster, and not be hardcoded by the Operator. Starting with this release, the default value is `crun`. (link:https://issues.redhat.com/browse/OCPBUGS-45450[*OCPBUGS-45450*])
```

Comment:
Could try to avoid future tense. Maybe:

```suggestion
The default container runtime that is set by the cluster Node Tuning Operator is always inherited from the cluster, and is not hard-coded by the Operator. Starting with this release, the default value is `crun`. (link:https://issues.redhat.com/browse/OCPBUGS-45450[*OCPBUGS-45450*])
```